package/scie-pants.lift.json

```diff
@@ -99,6 +99,18 @@
               "{scie.bindings.configure:PANTS_SHA_FIND_LINKS}"
             ]
           },
+          "pants-maybe-native-client": {
+            "description": "Runs a hermetic Pants installation via the native client (falling back to the legacy client if the native client is not available).",
+            "env": {
+              "=PANTS_VERSION": "{scie.bindings.configure:PANTS_VERSION}",
+              "PANTS_BUILDROOT_OVERRIDE": "{scie.bindings.configure:PANTS_BUILDROOT_OVERRIDE}",
+              "=_PANTS_SERVER_EXE": "{scie.bindings.install:VIRTUAL_ENV}/bin/pants"
```

Comment:
It seems like keeping the full value opaque and dried up in an install binding would be better. You calculate it there as well.

If the native client handled `PANTS_NO_NATIVE_CLIENT` processing as mentioned above, then the whole thing could just be handled by something like this change to the existing "pants" command:
```json
"pants": {
  "description": "Runs a hermetic Pants installation.",
  "env": {
    "=PANTS_VERSION": "{scie.bindings.configure:PANTS_VERSION}",
    "PANTS_BUILDROOT_OVERRIDE": "{scie.bindings.configure:PANTS_BUILDROOT_OVERRIDE}",
    "=_PANTS_SERVER_EXE": "{scie.bindings.install:PANTS_CONSOLE_SCRIPT}"
  },
  "exe": "{scie.bindings.install:PANTS_EXE}",
  "args": [
    "{scie.bindings.configure:PANTS_SHA_FIND_LINKS}"
  ]
},
```